enterprise/management/commands/monthly_impact_report.py

```diff
@@ -927,7 +927,7 @@ def get_query_results_from_snowflake(self):
             cs.execute(QUERY)
             rows = cs.fetchall()
             for row in rows:
-                yield row
+                yield row   # pylint: disable=use-yield-from,useless-suppression
```

Comment:
This warning has been added in `pylintrc` so the suppression is not needed anymore. 
```suggestion
                yield row
```

Reply:
It was giving warning `I0021: Useless suppression of 'use-yield-from' (useless-suppression)`.

Reply:
Since we've added it in pylintrc https://github.com/openedx/edx-enterprise/pull/2033/files#diff-f92a1a49c366f1f6086eace13a997b80bae46be0d72c920cb56701183a9d5c1cR294, it won't generate a warning now.